h/search/query.py

```diff
@@ -111,6 +106,20 @@ def extract_sort(params):
     }]
 
 
+def extract_search_after(params):
+    search_after = params.pop("start", None)
+    if search_after:
+        try:
+            return [float(search_after)]
```

Comment:
This assumes the field we are sorting by is a date which it isn't necessarily so that's gotta be fixed.